model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
```

Comment:
Can we explicitly name this as `Prometheus Exporters`. This way it's clear what we mean. You can also use id `prometheus-exporters` to be clear as well..

Reply:
OK I am confused now, we do have prometheus-exporters down, what's the deal with this one?

Reply:
What is the normal Prometheus agent? Can you send me a link here? Because I was not aware there is a Java agent for Prometheus?

Reply:
Yes sure => https://prometheus.io/docs/instrumenting/clientlibs/

Reply:
ah great, I was not aware of this.. should we call it `Prometheus ClientLibs`?